Source/missiles.cpp

```diff
@@ -52,22 +52,18 @@ void GetDamageAmt(int i, int *mind, int *maxd)
 		if (plr[myplr]._pClass == HeroClass::Warrior || plr[myplr]._pClass == HeroClass::Monk || plr[myplr]._pClass == HeroClass::Barbarian) {
 			*maxd <<= 1;
 		} else if (plr[myplr]._pClass == HeroClass::Rogue || plr[myplr]._pClass == HeroClass::Bard) {
-			*maxd += *maxd >> 1;
+			*maxd += *maxd / 2;
 		}
-		*mind = -1;

```

Comment:
Did you mean to delete this and `case SPL_LIGHTNING:`?

Reply:
No, I reverted that.